src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/main.scss"
     />
```

Comment:
Stylesheets with a .scss extension should not be linked directly in HTML. Instead, compile them to CSS and link the resulting .css file.